awx/api/serializers.py

```diff
@@ -2183,8 +2257,6 @@ def validate_credential_type(self, credential_type):
             for rel in (
                 'ad_hoc_commands',
                 'insights_inventories',
-                'inventorysources',
```

Comment:
If there's not an explicit test for this scenario, can we add one to make our lives easier when 3.2.3 merges (I want to make sure we don't regress on that merge)?